src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -0,0 +1,189 @@
+@import '~igz-controls/scss/shadows';
+@import '~igz-controls/scss/colors';
+
+.metrics {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  height: fit-content;
+
+  &__card-tmp {
+    z-index: 1;
+    top: 0;
+    position: sticky !important;
+    box-sizing: border-box;
+    border: 1px solid #00000033;
+    border-radius: 12px !important;
+    background: #fff;
+    padding-top: 12px !important;
+    width: 100%;
+    transition: 0.1s;
+    animation: ease-in-out;
+    max-height: 140px;
+  }
+
+  &__card {
+    border-radius: 12px;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+    padding: 6px 16px 35px;
+    transition: ease-in-out;
+    width: 100%;
+
+    &-drift-status {
+      margin-left: 8px;
+      border-radius: 50%;
+      height: 10px;
+      width: 10px;
+      display: inline-block;
+    }
+
+    &-header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+      gap: 8px;
+      padding-bottom: 10px;
+      padding-left: 5px;
+      font-weight: 500;
+
+      &-result-kind {
+        display: flex;
+        width: max-content;
+        justify-content: center;
+        align-self: center;
+        gap: 4px;
+        overflow: hidden;
+        color: #7f7989;
+        text-overflow: ellipsis;
+        font-family: Roboto;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 500;
+        line-height: normal;
+      }
+
+      &-data {
+        color: #7f7989;
```

Comment:
We never use colors directly. You should user color variables instead

Reply:
done